src/analytics/formChangeTracker/formChangeTracker.ts

```diff
@@ -55,8 +70,24 @@ export class FormChangeTracker extends FormTracker {
     }
   }
 
-  getSubmitterText(): string {
-    const submitButton = document.querySelector("form > button");
-    return submitButton?.textContent?.trim() ?? "undefined";
+  getSection(element: HTMLElement): string {
+    const parentElement = element.parentElement;
+
+    // Ensure the parent element exists
+    if (!parentElement) {
+      return "undefined";
+    }
+
+    let siblingElement = parentElement?.previousElementSibling;
```

Comment:
Can we rename `siblingElement` to `parentElementSibling` for readability?

Reply:
done